backend/editor/entries.py

```diff
@@ -173,13 +177,28 @@ def update_nodes(label, entry, new_node_keys):
             continue
         query.append(f"""\nREMOVE n.{key}\n""")
 
+    #Normalising Node to be Updated
+    normalised_new_node_keys = {}
+    for keys in new_node_keys.keys():
+        if keys.startswith("tags") and not keys.endswith("str"):
+            if "ids" not in keys:
+                keys_language_code = keys.split('_', 1)[1]
+                normalised_value = []
+                for values in new_node_keys[keys]:
+                    normalised_value.append(normalizing(values, keys_language_code))
+                normalised_new_node_keys[keys] = new_node_keys[keys]
+
+                normalised_new_node_keys["tags_ids_"+keys_language_code] = normalised_value

```

Comment:
I would add a else to make explicit it's intended
```suggestion
                normalised_new_node_keys["tags_ids_"+keys_language_code] = normalised_value
            else:
                pass  # we generate tags_ids, and ignore the one sent

```